src/Commands/pm/SecurityUpdateCommands.php

```diff
@@ -44,7 +45,7 @@ public static function composerLockPath(): string
      *
      * @command pm:security
      * @aliases sec,pm-security
-     * @bootstrap none
+     * @bootstrap configuration
```

Comment:
Unfortunately this is a regression. One use case is to run this in in cron/CI without an installed site present. The only prerequisite is composer.lock.

Reply:
I guess this only adds a settings.php requirement which isn't terrible but I'm not sure its a net positive.

Reply:
I tested it with a single composer.lock and didn't get any errors or warnings, I guess it could go wrong if Drupal is broken and could not bootstrap (but at this point you got bigger problems).

If you still find a regression about the bootstrap part, this dependency could be removed, the `getenv('https_proxy)'` part is still working.

Reply:
So, will you merge it or I have something to change ?

Reply:
You can change to `@bootstrap max`. That will go as far as possible, and then start running the command. Please test with an unconfigured site, just to make sure the command still work (of couse no proxy support in this case).

Reply:
I tried with a composer.lock alone and it works, should I write a test ?

Reply:
```bash
$ env |grep proxy
http_proxy=my-proxy:80
https_proxy=my-proxy:80
no_proxy=localhost,127.0.0.0,
$ mv  sites/default/settings.php sites/default/settings.old
$ drush sec
 [warning] One or more of your dependencies has an outstanding security update.
 [notice] Try running: composer require drupal/core --update-with-dependencies
 [notice] If that fails due to a conflict then you must update one or more root dependencies.
+-------------+-------------------+
| Name        | Installed Version |
+-------------+-------------------+
| drupal/core | 8.8.2             |
+-------------+-------------------+
$ unset http_proxy
$ unset https_proxy
$ drush sec
 [warning] file_get_contents(): php_network_getaddresses: getaddrinfo failed: Name or service not known SecurityUpdateCommands.php:136
 [warning] file_get_contents(https://raw.githubusercontent.com/drupal-composer/drupal-security-advisories/8.x-v2/composer.json): failed to open stream: php_network_getaddresses: getaddrinfo failed: Name or service not known SecurityUpdateCommands.php:136

In SecurityUpdateCommands.php line 141:

  Unable to fetch drupal-security-advisories information.


pm:security [--format FORMAT] [--fields FIELDS] [--field FIELD] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-d|--debug] [-y|--yes] [--no] [--remote-host REMOTE-HOST] [--remote-user REMOTE-USER] [-r|--root ROOT] [-l|--uri URI] [--simulate] [--pipe] [-D|--define DEFINE] [--filter [FILTER]] [--notify [NOTIFY]] [--druplicon] [--xh-link XH-LINK] [--] <command>
```